app/javascript/packages/verify-flow/steps/personal-key/personal-key-step.tsx

```diff
@@ -1,93 +0,0 @@
-import { useContext } from 'react';
-import { Alert, PageHeading } from '@18f/identity-components';
-import { ClipboardButton } from '@18f/identity-clipboard-button';
-import { PrintButton } from '@18f/identity-print-button';
```

Comment:
I think the React implementation of these two components would now be unused, and we could choose to remove them if we wanted (still need the JavaScript custom element and ViewComponent implementation though).

Reply:
Alright, axed in https://github.com/18F/identity-idp/pull/7051/commits/8068254a18638f6fb4318844cd27c679110499fc